ci/cron/src/Main.hs

```diff
@@ -168,6 +168,14 @@ build_docs_folder path versions latest = do
         copy from to = do
             shell_ $ "cp -r " <> from <> " " <> to
         build version path = do
+            case version of
+              "0.13.43" -> do
+                  shell_ "git checkout 0c4f9d7f92c4f2f7e2a75a0d85db02e20cbb497b"
+                  robustly_download_nix_packages
```

Comment:
Also we should add a comment that explains why we need this. I’m definitely going to forget this :slightly_smiling_face: